src/common/options.cc

```diff
@@ -6899,6 +6899,10 @@ std::vector<Option> get_mds_options() {
     .set_default(5)
     .set_description("minimum interval between scatter lock updates"),
 
+    Option("mds_client_msg_throttle", Option::TYPE_SIZE, Option::LEVEL_ADVANCED)
+    .set_default(1<<16)
+    .set_description("number of bytes client messages can consume in memory before being throttled"),
```

Comment:
Can also set it as ```.related_to``` ms_dispach_throttle.

Reply:
I don't see a `.related_to` method?

Reply:
`add_see_also`?

Reply:
Yeah I guess that one, sorry.

Reply:
Also note the set_policy_throttler() is a *byte* throttler.. need to update this description!